package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dhis2/analytics",
-    "version": "26.3.0",
+    "version": "999.9.9-TE-alpha.3",
```

Comment:
Should this change be undone?

Reply:
This should indeed be undone yeah, we'll do that just before releasing.